### ISSUE TITLE
Existing Managed Properties are not being Updated while we apply DSC Configurations

### ISSUE THREAD

**Issue by nikegambhava** [microsoft/Microsoft365DSC#2881]
Not able to Update managed properties using DSC 

i want to Update some Managed properties using dsc module 

when i apply some configurations for managed properties it is not overwriting or updating existing managed properties configurations

Current configuration is something like this
   SPOSearchManagedProperty e752a4cc-1677-4008-bd92-831108aa922e
        {
            AllowMultipleValues         = $False;
            Aliases = "NewNG"
            CompanyNameExtraction       = $False;
            CompleteMatching            = $False;
            Credential                  = $Credscredential;
            Description                 = "";
            Ensure                      = "Present";
            FinerQueryTokenization      = $False;
            FullTextContext             = 0;
            FullTextIndex               = "";
            LanguageNeutralTokenization = $False;
            MappedCrawledProperties     = @("ows_Name");
            Name                        = "NGNew";
            Queryable                   = $False;
            Refinable                   = "No";
            Retrievable                 = $True;
            Safe                        = $True;
            Searchable                  = $True;
            Sortable                    = "No";
            TokenNormalization          = $False;
            Type                        = "Text";
        }
Here i'm not able to change value for such check box fields like (Queryable,TokenNormalization etc..)

Also Crawl properties are not being mapped can someone help me here.

**Comment by NikCharlebois** [comment]
Indeed, it appears that the Set-PnPConfiguration cmdlet being used by M365DSC isn't updating the fields anymore. I will open an issue with the PnP team.

**Comment by NikCharlebois** [comment]
https://github.com/pnp/powershell/issues/2803
